src/runtime/node/internal/perf_hooks/performance.ts

```diff
@@ -37,11 +33,123 @@ const nodeTiming = {
   },
 } satisfies nodePerfHooks.PerformanceNodeTiming;
 
-// Performance
-export const Performance = class Performance
-  extends _Performance<nodePerfHooks.PerformanceEntry>
-  implements nodePerfHooks.Performance
+// PerformanceEntry
+export class PerformanceEntry implements nodePerfHooks.PerformanceEntry {
+  readonly __unenv__ = true;
+  detail: any | undefined;
+  entryType = "event" as any; // abstract
+  name: string;
+  startTime: number;
+  constructor(name: string, options?: PerformanceMarkOptions) {
+    this.name = name;
+    this.startTime = options?.startTime || performance.now();
```

Comment:
Should this fallbacks to Date.now and use globalThis.performance?.now() (same as Performance#now)

Reply:
https://github.com/unjs/unenv/pull/478/commits/c916adc68d20d27e0226e2cc3fd656e43a88fe96

Reply:
or better applied same change of #478